misc/python/materialize/parallel_workload/action.py

```diff
@@ -1286,6 +1355,7 @@ def __init__(
 
 ddl_action_list = ActionList(
     [
+        (SleepAction, 100),  # DDLs easily starve Mz, don't run them too often
```

Comment:
Will this cause too many SQL connections to be actually sleeping at any given time, vs. running DMLs?

You may wish to tackle the imbalance not via `SleepAction`, but by changing the ratio each of the `_list`s is being picked.

Reply:
No, each connection already only runs DMLs or DDLs, never both. This affects only the DDL connections. I noticed that they are kind of drowing out the DML and read connections.